examples/platforms/nrf528xx/nrf52840/openthread-core-nrf52840-config.h

```diff
@@ -285,7 +285,34 @@
  *
  */
 #ifndef OPENTHREAD_CONFIG_CSL_SAMPLE_WINDOW
-#define OPENTHREAD_CONFIG_CSL_SAMPLE_WINDOW 30
+#define OPENTHREAD_CONFIG_CSL_SAMPLE_WINDOW 10
+#endif
+
+/**
+ * @def OPENTHREAD_CONFIG_PLATFORM_DELAY_TRANSMIT_SUPPORTED
+ *
+ * The platform radio has the ability to transmit after a specified delay time.
+ *
+ */
+#define OPENTHREAD_CONFIG_PLATFORM_DELAY_TRANSMIT_SUPPORTED 1
```

Comment:
Suggest to remove this define and let radio return capability `OT_RADIO_CAPS_TRANSMIT_TIMING`